alacritty/src/daemon.rs

```diff
@@ -15,10 +15,10 @@ use winapi::um::winbase::{CREATE_NEW_PROCESS_GROUP, CREATE_NO_WINDOW};
 /// Start the daemon and log error on failure.
 pub fn start_daemon<I, S>(program: &str, args: I)
 where
-    I: IntoIterator<Item = S> + Debug + Copy,
+    I: IntoIterator<Item = S> + Debug + Clone,
     S: AsRef<OsStr>,
 {
-    match spawn_daemon(program, args) {
+    match spawn_daemon(program, args.clone()) {
```

Comment:
The args is used below for logging. I want to pass iterator `args.chain(iter::once(target))` to avoid `Vec`, but unfortunately chained iterator is not `Copy`. 
To reduce complexity, I revert this part back and keep the code of collecting `Vec`.